src/_data/sites/hi.yml

```diff
@@ -0,0 +1,521 @@
+  placement: ""
+
+#------------------------------------------------------------------------------
+# Shared
+#------------------------------------------------------------------------------
+
+shared:
+  get_started: शुरू करें
+  become_a_sponsor: एक प्रायोजक बनें
+  eslint_logo_alt: ESLint प्रतीक चिन्ह
+  description: >
+    JavaScript में पैटर्न को पहचानने और रिपोर्ट करने के लिए प्लग करने योग्य और कॉन्फ़िगर करने योग्य लिंटर टूल। अपने कोड की गुणवत्ता को आसानी से बनाए रखें।
+  title_format: PAGE_TITLE - ESLint - Pluggable JavaScript Linter
+  skip_to_content: मुख्य विषयवस्तु में जाएं
+
+#------------------------------------------------------------------------------
+# Navigation
+#------------------------------------------------------------------------------
+
+navigation:
+  - text: टीम
+    link: team
+  - text: ब्लॉग
+    link: blog
+  - text: दस्तावेज़
+    link: docs
+  - text: दुकान
+    link: store
+    target: _blank
+  - text: Playground
+    link: playground
+    target: _blank
+
+#------------------------------------------------------------------------------
+# Footer
+#------------------------------------------------------------------------------
+
+footer:
+  title: अपना JavaScript कोड ठीक करने के लिए तैयार हैं?
+  description: npm से इंस्टॉल करें या आज ही दान करना शुरू करें।
+  secondary: Secondary
+  social_icons:
+    title: सामाजिक मीडिया
+    twitter: Twitter
+    chat: Discord
+    mailing_list: Google Group
+    github: GitHub
+  theme_switcher:
+    title: Theme Switcher
+    light: Light
+    dark: Dark
+  language_switcher:
+    title: Language Switcher
+    description: एक भाषा का चयन आपको उस भाषा में ESLint वेबसाइट पर ले जाएगा।
+    change_language: भाषा बदलें
+    language: भाषा
+  copyright: >
+    Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and ESLint 
+    contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS 
+    Foundation</a> has registered trademarks and uses trademarks.  For a list of 
+    trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please 
+    see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> 
+    and <a href="https://trademark-list.openjsf.org">Trademark List</a>.  
+    Trademarks and logos not indicated on the 
+    <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation 
+    trademarks</a> are trademarks&trade; or registered&reg; trademarks of 
+    their respective holders. Use of them does not imply any affiliation with 
+    or endorsement by them.
+  links:
+    open_jsf: The OpenJS Foundation
+    terms: उपयोग की शर्तें
+    privacy: गोपनीयता नीति
+    bylaws: OpenJS Foundation Bylaws
+    trademark: Trademark नीति
+    trademark_list: Trademark सूची
+    cookies: Cookie नीति
+
+#------------------------------------------------------------------------------
+# 404 Page
+#------------------------------------------------------------------------------
+
+404_page:
+  title: 404 ग़लती
```

Comment:
@snitin315 - How about keeping it something like "404 पृष्ठ नहीं मिला" ?

Reply:
Sounds good to me.

Reply:
But we already have a subtitle with `पृष्ठ नहीं मिला`. So it would be double. Or we can just keep `404` only.

<img width="564" alt="Screenshot 2022-05-03 at 1 47 31 PM" src="https://user-images.githubusercontent.com/46647141/166423016-599447e5-eed7-4189-bf8b-3e2d7dd03ced.png">